autocomplete.js

```diff
@@ -0,0 +1,71 @@
+const _ = require("lodash");
+const parsers = require("./parsers");
+const consts = require("./consts.json");
+
+/**
+ * @param params: [ { value: any, name: string, type: string } ]
+ * Array of values passed to autocomplete function as action parameters.
+ */
+function mapAutocompleteFuncParamsToObject(params) {
```

Comment:
Please order the functions from the outermost to innermost, maintaining the export order.